drivers/SmartThings/matter-switch/fingerprints.yml

```
@@ -71,6 +71,12 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+  - id: "5020/43521"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA01
+    deviceProfileName: "switch-binary"
```

```suggestion
    deviceProfileName: switch-binary
```
I don't think this will cause issues with quotes, but for consistency, all other fingerprints are formatted without quotes around the `deviceProfileName`